package.json

```diff
@@ -25,32 +42,50 @@
     "@babel/preset-env": "^7.1.0",
     "@babel/preset-react": "^7.0.0",
     "babel-core": "^7.0.0-bridge.0",
+    "babel-eslint": "^10.0.1",
     "babel-jest": "^23.6.0",
     "babel-loader": "^8.0.4",
     "enzyme": "^3.7.0",
     "enzyme-adapter-react-16": "^1.6.0",
     "eslint": "^5.7.0",
     "eslint-config-airbnb": "^17.1.0",
+    "eslint-config-prettier": "^3.1.0",
     "eslint-plugin-import": "^2.14.0",
     "eslint-plugin-jsx-a11y": "^6.1.2",
+    "eslint-plugin-prettier": "^3.0.0",
     "eslint-plugin-react": "^7.11.1",
     "faker": "^4.1.0",
+    "husky": "^1.1.2",
     "install": "^0.12.2",
     "jest": "^23.6.0",
+    "jest-junit": "^5.2.0",
+    "lint-staged": "^8.0.0",
     "npm": "^6.4.1",
+    "prettier": "^1.14.3",
     "react-test-render": "^1.1.1",
     "regenerator-runtime": "^0.12.1",
-    "webpack": "^4.23.0",
-    "webpack-cli": "^3.1.2"
+    "webpack-cli": "^3.1.2",
+    "webpack-dev-server": "^3.1.10"
   },
   "dependencies": {
```

Comment:
Could some of these be devDependencies? (I'm thinking of the webpack ones, mostly)

Reply:
I believe I had these in my devDependencies before and eslint made me put them in dependencies.